set-web/src/main/java/eu/europeana/set/web/service/controller/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package eu.europeana.set.web.service.controller.exception;
+
+import javax.annotation.Resource;
+
+import org.springframework.web.bind.annotation.ControllerAdvice;
+
+import eu.europeana.api.commons.config.i18n.I18nService;
+import eu.europeana.api.commons.web.controller.exception.AbstractExceptionHandlingController;
+import eu.europeana.api.commons.web.model.ApiResponse;
+import eu.europeana.set.web.model.UserSetOperationResponse;
+
+@ControllerAdvice
+public class GlobalExceptionHandler extends AbstractExceptionHandlingController {
+
+	@Resource
+	I18nService i18nService;
+
+	protected I18nService getI18nService() {
+		return i18nService;
+	}
+	
+	@Override
+	public ApiResponse buildErrorResponse(String errorMessage, String action, String apiKey) {
```

Comment:
Why is this method needed? There is an buildErrorResponsemethod in the paren implementation

Reply:
removed as suggested